app/src/main/java/org/simple/clinic/home/patients/illustration/DayOfMonth.kt

```diff
@@ -0,0 +1,30 @@
+package org.simple.clinic.home.patients.illustration
+
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.ToJson
+import org.simple.clinic.util.RoomEnumTypeConverter
+import org.threeten.bp.Month
+import java.util.Locale
+
+data class DayOfMonth(
+    val day: Int,
+    val month: Month
+) {
+
+  class MonthRoomTypeConverter : RoomEnumTypeConverter<Month>(Month::class.java)
+
+  object MoshiTypeConverter {
```

Comment:
Could you add a test for this class please?

Reply:
done